### PR TITLE
Added the option to change password directly through chromebot

### LINKED ISSUE #220: To create a feature allowing to change password directly through the chromebot
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/susi_server

**Current behavior:**
Only being able to change the password via the main app

**Expected behavior:**
To be able to change password from the chromebot

**Related code:**

Using susi api and jquery 

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by stealthanthrax**
Fixes #220 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
This adds a feature to change password directly through chromebot

#### Changes proposed in this pull request:

34494754-e95e2592-f017-11e7-8bda-89c508dbfd05

### CODE REVIEW

**Comment by stealthanthrax** [comment]
@ms10398 @gabru-md @zamhaq @akshatnitd @arnav-mandal1234 @VishalSharma0303 ,this PR is made because the previous PR was beyond repair

File: src/js/login.js [hunk 1]
```diff
@@ -29,6 +34,34 @@ document.getElementById("toggle").addEventListener("click", ()=>{
...
 
+pass.addEventListener("click", ()=>{
+$("#cPass").toggle();
+});
+
+
+cPass.addEventListener("submit", (e)=>{
+	e.stopPropagation();
+							
+		    var loginEP = BASE_URL+"/aaa/changepassword.json?"+"changepassword="+ cemail.value + "&password="+ cpassword.value + "&newpassword=" + newPassword.value + "&access_token=" + accessToken ;
```
Comment (gabru-md) [inline src/js/login.js]: Please check for indentation here.

**Comment by stealthanthrax** [comment]
@gabru-md @ms10398 @zamhaq , have a look again

**Comment by stealthanthrax** [comment]
@gabru-md @ms10398

File: src/js/login.js [hunk 2]
```diff
@@ -29,6 +34,31 @@ document.getElementById("toggle").addEventListener("click", ()=>{
...
+
+cPass.addEventListener("submit", (e)=>{
+	e.stopPropagation();
+		var loginEP = BASE_URL+"/aaa/changepassword.json?"+"changepassword="+ cemail.value + "&password="+ cpassword.value + "&newpassword=" + newPassword.value + "&access_token=" + accessToken ;		    
+		  $.ajax({
+		    	url:loginEP,
+		        dataType: "jsonp",
+				jsonp: "callback",
+				crossDomain: true,
+				crossDomain: true,
```
Comment (gabru-md) [inline src/js/login.js]: `crossDomain` twice?

File: src/login.html [hunk 3]
```diff
@@ -52,7 +52,20 @@
                     <div class="loggedin" id="loggedin">
                         <button class="btn btn-success" name="Logout" onClick="logout(this)" id="logout">Logout</button>
                         <br />
+                        <a  style="margin-left:35%; color:#fff;"  id="pass" href="#" >Click here to change password</a>
                     </div>
+                     <br><br>
+                    <div style="margin-left: 27%;">
+                    <form id="cPass" style="display:none;">
+                        <input type="email" id="cemail" placeholder="Email">
+                        <br><br>
```
Comment (gabru-md) [inline src/login.html]: @stealthanthrax Instead of using `br` can we use `margin` ?

**Comment by stealthanthrax** [comment]
done @gabru-md

**Pull request merged**